pkgs/tools/system/xmousepasteblock/default.nix

```diff
@@ -0,0 +1,30 @@
+{ xorg, stdenv, fetchgit, libev, pkgconfig }:
+
+stdenv.mkDerivation {
+  name = "XMousePasteBlock-1.0";
+  src = fetchgit {
+    sha256 = "0vidckfp277cg2gsww8a8q5b18m10iy4ppyp2qipr89771nrcmns";
+    url = "https://github.com/milaq/XMousePasteBlock";
+    rev = "ef7887d75a38628b7c7990c426297beeb639b706";
```

Comment:
You can use the tag `1.0` instead.

Reply:
I am not sure I can specify tags with fetchFromGithub